waku/v2/node/config.nim

```diff
@@ -129,6 +130,34 @@ type
       defaultValue: "/toy-chat/2/luzhou/proto"
       name: "rln-relay-content-topic" }: ContentTopic
     
+    rlnRelayDynamic* {.
```

Comment:
Perhaps not yet for this PR, but a good case for why we need to start using sub-commands as explained in https://github.com/status-im/nim-confutils to group related commands together :)

Reply:
Sure, sounds a good idea, much cleaner and organized! here is the tracking issue https://github.com/status-im/nwaku/issues/999